prawns/governance/src/handlers/democracy.NotPassed.event.ts

```diff
@@ -0,0 +1,27 @@
+import { EventHandlerContext } from '@subsquid/substrate-processor';
+import { SubstrateNetwork } from '../model';
+import { getDemocracyNotPassedEvent } from './typeGetters/getDemocracyNotPassedEvent';
+import substrateDemocracyReferendaRepository from '../repositories/substrateDemocracyReferendaRepository';
+
+
+export default (network: SubstrateNetwork) =>
+  async (ctx: EventHandlerContext) => {
+    if (!ctx.event) {
+      return;
+    }
+
+    const date = new Date(ctx.block.timestamp);
+    const event = getDemocracyNotPassedEvent(ctx, network);
+
+    const referenda = await substrateDemocracyReferendaRepository.getByReferendaIndex(ctx, network, event.refIndex);
+
+    if (!referenda) {
+      throw new Error(`Referenda not found`);
+    }
+
+    referenda.status = 'not_passed';
```

Comment:
Sorry, didn't notice when you built the schema but one idea could be:
Add this to the governance schema:
```
enum SubstrateDemocracyReferendaStatus {
  passed
  notPassed
}

(...)
type SubstrateDemocracyReferenda @entity {
  status: SubstrateDemocracyReferendaStatus!
```

Then in the code you could do:
```
import { SubstrateDemocracyReferendaStatus } from '../model';

(...)

referenda.status = SubstrateDemocracyReferendaStatus.passed;
```

This would be particularly interesting if we are doing filtering, as graphql shows you the two options.
In the end is just a suggestion/improvement, I can understand if you don't wanna take it on as it's a bit of extra work :D

Reply:
You are right... they should be enums 👍